Client/src/App/DataRestriction/RestrictedComponent.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { ReactEventHandler, ReactNode } from 'react';
+
+import { NotFoundController } from '@veupathdb/wdk-client/lib/Controllers';
+
+import { ApprovalStatus } from 'ebrc-client/hooks/dataRestriction';
+
+interface Props {
+  approvalStatus: ApprovalStatus;
+  children: ReactNode;
+}
+
+export function RestrictedComponent({
```

Comment:
It might be nice to add some documentation here

Reply:
Definitely! In addition, I think renaming the component to something like `RestrictedPage` would go a long way towards communicating the intended use.